include/d/a/obj/d_a_obj_base.h

```diff
@@ -32,7 +32,11 @@ struct ActorCarryStruct {
     /* 0x90 */ f32 field_0x90;
     /* 0x94 */ f32 field_0x94;
     /* 0x98 */ f32 field_0x98;
-    /* 0x9C */ void *dtor; // ???
+
+    ActorCarryStruct();
+    /* vt 0x9C */
+    virtual ~ActorCarryStruct();
+    // Is this actually part of this struct?
     /* 0xA0 */ u32 field_0xA0;
     /* 0xA4 */ u32 field_0xA4;
```

Comment:
maybe, I recall needing it at some point. I was carried related (arrow used part of it, loftwing A4 is an actor reference, loftwing used it when carrying player), A0 is a reference to to whatever fn_80358660 constructs